ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/amd_operator.sh

```diff
@@ -132,5 +131,13 @@ echo "Installing AMD operator"
 oc apply -f "$GPU_INSTALL_DIR/amd_gpu_install.yaml"
 wait_while 360 ! has_csv_succeeded openshift-amd-gpu amd-gpu-operator
 create_devconfig
-name=$(oc get pod -n openshift-amd-gpu -l openshift.io/build.name -oname)
-wait_while 1200 ! monitor_logs "$name" "Successfully pushed image-registry.openshift-image-registry.svc:5000/openshift-amd-gpu" openshift-amd-gpu docker-build
+wait_until_pod_is_created  openshift.io/build.name openshift-amd-gpu 180
+if [[ $? -eq 0 ]];
+  then
+    name=$(oc get pod -n openshift-amd-gpu -l openshift.io/build.name -oname)
+    echo Builder pod name: $name
+    oc wait --timeout="60s" --for=condition=ready pod -n openshift-amd-gpu -l openshift.io/build.name
+  else 
+    exit 1
+fi
+wait_while 1200 monitor_logs "$name" openshift-amd-gpu docker-build "Successfully pushed image-registry.openshift-image-registry.svc:5000/openshift-amd-gpu"
```

Comment:
still not working well, it happened that after the builder pod finished, it got terminated and the log checker didn't find the string and kept checking until timeout

```
18:29:49  .Monitoring logs for pod pod/dc-internal-registry-build-d4g97-build...
18:29:49  Error from server (NotFound): pods "dc-internal-registry-build-d4g97-build" not found
18:29:52  .Monitoring logs for pod pod/dc-internal-registry-build-d4g97-build...
```